src/components/flowchart/flowchart.test.js

```diff
@@ -88,10 +88,9 @@ describe('FlowChart', () => {
     const instance = wrapper.find('FlowChart').instance();
     const viewExtents = getViewExtents(instance.view);
 
-    expect(viewExtents).toEqual({
-      translate: { minX: -500, minY: -500, maxX: 1044, maxY: 1553 },
-      scale: { minK: 0.8, maxK: 2 },
-    });
+    expect(viewExtents.translate).toEqual({ minX: -500, minY: -500, maxX: 1044, maxY: 1553 });
+    expect(viewExtents.scale.minK).toBeCloseTo(0.777);
```

Comment:
What is the value `0.777` represent? This would be more intuitive if this is presented as a calculated value rather than an exact figure.

Reply:
That's fair I should have mentioned these tests were just an integration check for sanity, partly to avoid testing implementation / re-implementing the tested function in the test itself, but I've managed to wrangle it into something fairly reasonable so please check out the updated version.